language/polkavm/move-to-polka/tests/examples_basic_morebasic_move_test.rs

```diff
@@ -0,0 +1,29 @@
+use polkavm::{Config, Engine, Linker, Module, ProgramBlob};
+
+#[test]
+pub fn test_program_execution() {
+    env_logger::init();
+    //TODO(tadas): this should be replaced with full compilation process from move sources, polka linker etc.
+    let program_bytes = include_bytes!("../../../../output/morebasic.polkavm");
```

Comment:
Personally, I like the fact that we have some `.polkavm` files as test fixtures. So we can have a "small" test like this for a limited functionality.  Or maybe this could be a reusable test function where we just need a a polkavm file or bytes. I guess the only input is the bytes. 

But I can also see that this test is useless if we just use a `.polkavm` file as  fixture. Because we are only testing polkavm functionality. Nothing from our crate.